services/src/main/java/org/keycloak/broker/saml/SAMLIdentityProviderFactory.java

```diff
@@ -164,6 +171,52 @@ public SAMLIdentityProviderConfig createConfig() {
+                        // import attributes only if values with these locale exist
+                        uiInfo.getDisplayName().stream().filter(dn -> lang.equals(dn.getLang())).findFirst().ifPresent(
+                            displayName -> samlIdentityProviderConfig.setConfigMduiDisplayName(displayName.getValue()));
+                        uiInfo.getDescription().stream().filter(dn -> lang.equals(dn.getLang())).findFirst()
+                            .ifPresent(description -> samlIdentityProviderConfig.setMduiDescription(description.getValue()));
+                        uiInfo.getInformationURL().stream().filter(dn -> lang.equals(dn.getLang())).findFirst()
+                            .ifPresent(informationURL -> samlIdentityProviderConfig
+                                .setMduiInformationURL(informationURL.getValue().toString()));
+                        uiInfo.getPrivacyStatementURL().stream().filter(dn -> lang.equals(dn.getLang())).findFirst()
+                            .ifPresent(privacyStatementURL -> samlIdentityProviderConfig
+                                .setMduiPrivacyStatementURL(privacyStatementURL.getValue().toString()));
+                        if (!uiInfo.getLogo().isEmpty()) {
+                            LogoType logo = uiInfo.getLogo().get(0);
+                            samlIdentityProviderConfig.setMduiLogo(logo.getValue().toString());
+                            samlIdentityProviderConfig.setMduiLogoHeight(logo.getHeight());
+                            samlIdentityProviderConfig.setMduiLogoWidth(logo.getWidth());
+                        }
+
+                    }
+
+                    // organization
+                    if (entityType.getOrganization() != null) {
+                        entityType.getOrganization().getOrganizationName().stream().filter(dn -> lang.equals(dn.getLang()))
+                            .findFirst().ifPresent(organizationName -> samlIdentityProviderConfig
+                                .setMdOrganizationName(organizationName.getValue()));
+                        entityType.getOrganization().getOrganizationDisplayName().stream()
+                            .filter(dn -> lang.equals(dn.getLang())).findFirst()
+                            .ifPresent(organizationDisplayName -> samlIdentityProviderConfig
+                                .setMdOrganizationDisplayName(organizationDisplayName.getValue()));
+                        entityType.getOrganization().getOrganizationURL().stream().filter(dn -> lang.equals(dn.getLang()))
+                            .findFirst().ifPresent(organizationURL -> samlIdentityProviderConfig
+                                .setMdOrganizationURL(organizationURL.getValue().toString()));
+                    }
+
+                    // contact person
+                    if (!entityType.getContactPerson().isEmpty()) {
+                        ContactType contact = entityType.getContactPerson().get(0);
```

Comment:
`ContactPerson` is a list of persons. It does not seem correct to store only a single person - either all or none should be stored.